pyccl/tests/test_pkhm.py

```diff
@@ -182,6 +186,44 @@ def test_pkhm_pk2d():
     assert np.all(np.fabs((pk_arr / pk_arr_2 - 1)).flatten()
                   < 1E-4)
 
+    # 1h/2h transition
+    def alpha0(a):  # no smoothing
+        return 1.
+
+    def alpha1(a):
+        return 0.7
+
+    pk0 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           smooth_transition=None)
+    pk1 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           smooth_transition=alpha0)
+    assert np.allclose(pk0, pk1, rtol=0)
+    pk2 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           smooth_transition=alpha1)
+    assert np.all(pk2/pk0 > 1.)
+
+    # 1-halo damping
+    def ks0(a):  # no damping
+        return 1e-16
+
+    def ks1(a):  # fully supressed
+        return 1e16
+
+    pk0 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           supress_1h=None, get_2h=False)
+    pk1 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           supress_1h=ks0, get_2h=False)
+    assert np.allclose(pk0, pk1, rtol=0)
+    pk2 = ccl.halos.halomod_power_spectrum(COSMO, hmc, k_arr, a_arr, P1,
+                                           normprof1=True, normprof2=True,
+                                           supress_1h=ks1, get_2h=False)
+    assert np.allclose(pk2, 0, rtol=0)
```

Comment:
Can you add one more that is completely quantitative? I.e. make k_star a sensible value (0.05 or whatever), and test that the resulting pk is equal to pk0 multiplied by the corresponding k-dependent factor.

Reply:
Done!